framework/src/main/java/org/checkerframework/framework/flow/CFAbstractStore.java

```diff
@@ -940,41 +940,47 @@ public int hashCode() {
     @SideEffectFree
     @Override
     public String toString() {
-        return "Use a CFGVisualizer to see the Store: " + this.hashCode();
+        return "Use visualize(new StringCFGVisualizer<>()) to see the Store: " + this.hashCode();
```

Comment:
Please replace this with
`return visualize(new StringCFGVisualizer<>(...))`
if possible.

Reply:
It works properly. Sorry I got it wrong at first, I have already changed it.

Reply:
If I understand correctly, now when we call `toString()`, we can get the String version of the value of this Store directly.